includes/admin/plugins.php

```diff
@@ -74,20 +88,41 @@ function perflab_render_plugins_ui() {
 	$experimental_plugins = array();
 
 	foreach ( perflab_get_standalone_plugin_data() as $plugin_slug => $plugin_data ) {
+		$api_data = perflab_query_plugin_info( $plugin_slug ); // Data from wordpress.org.
+
+		// Skip if the plugin is not on WordPress.org or there was a network error.
+		if ( $api_data instanceof WP_Error ) {
+			wp_admin_notice(
+				esc_html(
+					sprintf(
+						/* translators: %s is the plugin slug */
+						__( 'Failed to query WordPress.org Plugin Directory for plugin "%s".', 'performance-lab' ) . ' %s',
+						$plugin_slug,
+						$api_data->get_error_message()
```

Comment:
```suggestion
						/* translators: 1: plugin slug. 2: error message. */
						__( 'Failed to query WordPress.org Plugin Directory for plugin "%1$s". %2$s', 'performance-lab' ),
						$plugin_slug,
						$api_data->get_error_message()
```

Reply:
Actually I did it this way to reuse the core translation string.

Reply:
Oh no, that was somewhere else I guess.

Reply:
Oh yeah, it should do the same thing as I commented at https://github.com/WordPress/performance/pull/1175/files#r1576999210

Reply:
Meh. That's too generic. Nevermind.